### PR TITLE
[NO TICKET] Add icons to settings page

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
This PR aims to do the following improvements in the Demo app.

- Add icons to the buttons in the settings page
- Remove `Display Miniapp From List` button from `Display by URL` page to adjust with bottom navigation tray e.g. "List I", "List II" etc. Instead, there is a back arrow icon added to the top to go back to the previous page.

# Screenshots 
| Previous Design | Redesign by this PR |
----|---- 
| Screen Shot 2022-12-23 at 13 50 21 | Screen Shot 2022-12-23 at 13 49 26 |
| Screen Shot 2022-12-23 at 13 35 22 | Screen Shot 2022-12-23 at 13 31 38 |
| Screen Shot 2022-12-23 at 13 36 39 | Screen Shot 2022-12-23 at 13 32 16 |

# Checklist
- [x] I have read the [contributing guidelines](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md#sdk-development-learning-path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: testapp/src/main/res/layout/settings_fragment.xml [hunk 1]
```diff
@@ -2,250 +2,243 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools">
 
-    <RelativeLayout
```
Comment (khairul-alam-licon) [inline testapp/src/main/res/layout/settings_fragment.xml:5]: I have removed it because it's redundant.

**Review by shohiebsense — APPROVED** [review]
Apart from the URL Icon not my cup of tea, LGTM!

**Review by munir-rakuten — APPROVED** [review]
Looks good to me , thanks for your effort.👍

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/CustomButtonViewWithArrow.kt [hunk 2]
```diff
@@ -33,32 +33,42 @@ class CustomButtonViewWithArrow @JvmOverloads constructor(
...
-                DemoAppAnalytics.init(AppSettings.instance.projectIdForAnalytics).sendAnalytics(
-                    RATEvent(
-                        event = EventType.APPEAR,
-                        action = action,
-                        pageName = pageName,
-                        siteSection = siteSection,
-                        componentName = btnLabel,
-                        elementType = "ButtonWithTextArrow"
-                    )
-                )
```
Comment (rleojoseph) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/CustomButtonViewWithArrow.kt:59]: Any reason why we are removing this @khairul-alam-licon ?

**Review by rleojoseph — APPROVED** [review]
Excellent job @khairul-alam-licon . Thank you for the changes

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/CustomButtonViewWithArrow.kt (hunk 2 above)
Comment (khairul-alam-licon) [reply testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/CustomButtonViewWithArrow.kt:59]: I have moved this part outside of `context.theme.obtainStyledAttributes(attrs, R.styleable.RatCustomAttributes, 0, 0)
            .let {...}` as expected. We have done the same in other RAT view component e.g. `RATButton`. 
I've synced on this with @munir-rakuten also.

Comment (khairul-alam-licon) [reply testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/CustomButtonViewWithArrow.kt:59]: It has been moved to [CustomButtonViewWithArrow.kt#L62~L71](https://github.com/rakutentech/android-miniapp/blob/7ed6d33826b8d359fc008b1beff371e8849865a5/testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/CustomButtonViewWithArrow.kt#L62~L71) now.

**Pull request merged**